src/PointwiseFunctions/Hydro/TagsDeclarations.hpp

```diff
@@ -60,6 +60,21 @@ template <typename DataType>
 struct SpecificInternalEnergy;
 template <typename DataType, size_t Dim, typename Fr = Frame::Inertial>
 struct MassFlux;
+
+// All tags for primitive relativistic fluid quantities
+template <typename DataType, size_t Dim, typename Frame = Frame::Inertial>
+using all_relativistic_primitive =
```

Comment:
again these names don't make sense: the Lorentz factor isn't a "primitive" in any paper. It's a side effect of how the recovery schemes work. Furthermore, depending on the EoS you have more quantities listed here than are necessary.

Reply:
or not enough